reflex/components/overlay/menu.py

```diff
@@ -74,7 +74,13 @@ def get_event_triggers(self) -> dict[str, Union[Var, Any]]:
         }
 
     @classmethod
-    def create(cls, *children, button=None, items=None, **props) -> Component:
+    def create(
+        cls,
+        *children,
+        button: Optional[Component] = None,
+        items: Optional[List] = None,
```

Comment:
is it possible for `items` to be a `Var`? doesn't look like that case is handled

Reply:
It wasn't supported initially, should I add this in this PR, or I can add it in a separate one.

Reply:
i'm fine either way, i just anticipate that coming as the next question 😂

Reply:
We can make a new PR for this